kernel/configs/powerpc_kata_kvm_4.19.x

```diff
@@ -2895,7 +2901,7 @@ CONFIG_CRC4=y
 # CONFIG_CRC7 is not set
 # CONFIG_LIBCRC32C is not set
 # CONFIG_CRC8 is not set
-# CONFIG_RANDOM32_SELFTEST is not set
+CONFIG_RANDOM32_SELFTEST=y
```

Comment:
not sure if you want a selftest - we probably presume the host side virtio random generator or the hardware random generator works - and this might cut into your guest boot time.